CHANGELOG.md

```diff
@@ -1055,7 +1084,7 @@
 
 - [Patches] Support new XenServer Updates system. See [our documentation](https://xen-orchestra.com/docs/updater.html#xenserver-updates). (PR [#7044](https://github.com/vatesfr/xen-orchestra/pull/7044))
 - [Host/Advanced] New button to download system logs [#3968](https://github.com/vatesfr/xen-orchestra/issues/3968) (PR [#7048](https://github.com/vatesfr/xen-orchestra/pull/7048))
-- [Home/Hosts, Pools] Display host brand and version (PR [#7027](https://github.com/vatesfr/xen-orchestra/pull/7027))
```

Comment:
It seems that something automatically deleted a space on several lines of this file. Can you undo it?

Reply:
Indeed, that's a weird change, which appear multiple times :thinking:

Reply:
original changelog has some "non breaking spaces"  Alt +space ( char 160 ) 

We should clean them in this PR